game.py

```diff
@@ -0,0 +1,145 @@
+import pygame
+from gi.repository import Gtk
+from generator import Generator
```

Comment:
This gets imported in this commit but then the file gets added in a different commit, either move the addition here or move the usage to the addition.

Reply:
You could squash both commits as that's easier.

Adding your personal contact details to the license isn't a great idea, please remove it.

Reply:
Can you please tell me how can I squash ( [19986b6](https://github.com/sugarlabs/make-them-fall-activity/pull/25/commits/19986b60de562997dcf5821c806ff3bedf27efdd) and [44c92ca](https://github.com/sugarlabs/make-them-fall-activity/pull/25/commits/44c92cab94c42b5b31f171f6420e1d5a92a2259f) ) these two commits, I tried but it is getting real messy

Reply:
You can run an interactive rebase on the parent commit and in this case that's 363793c so it'll look something like this;
```bash
# This should open your editor and then you can look for both commits and move them close to each other
$ git rebase -i 363793c
# So in your editor you'll have something like this
pick 19986b6 Created generic Game class
pick efddd14 Create class Spike in spike.py
pick 44c92ca created Generator class in generator.py
# Which you'll rearrange to
pick 19986b6 Created generic Game class
s 44c92ca created Generator class in generator.py
pick efddd14 Create class Spike in spike.py
# Then exit the editor and you'll be taken into another editor to edit both commit messages and you can 
# then combine them into one and delete the other so you'll have one commit message and the 
# other would be empty then exit
$ git rebase --continue # should be the final trick.

Reply:
Thanks Ibiam, I should have asked beforehand, I spent a lot of time figuring this out while completely forgetting that I can change the order of the commits in the rebase 😅